source/common/res/features/pacing/main.js

```diff
@@ -80,97 +81,101 @@
+                ((ynabToolKit.l10nData && ynabToolKit.l10nData['toolkit.pacing']) || 'PACING') + '</li>');
 
-            var deemphasizedCategories = getDeemphasizedCategories();
+              var deemphasizedCategories = getDeemphasizedCategories();
 
-            var showIndicator = ynabToolKit.options.pacing;
-            if (showIndicator === '2') {
-              showIndicator = true;
-            } else {
-              showIndicator = false;
-            }
+              var showIndicator = ynabToolKit.options.pacing;
+              if (showIndicator === '2') {
+                showIndicator = true;
+              } else {
+                showIndicator = false;
+              }
 
-            // Select all budget table rows but not the uncategorized category and not master categories.
-            $('.budget-table-row')
-              .not('.budget-table-uncategorized-transactions')
-              .not('.is-master-category')
-              .each(function () {
-                var available = ynab.YNABSharedLib.defaultInstance.currencyFormatter.unformat($(this).find('.budget-table-cell-available').text());
-                var activity = -ynab.YNABSharedLib.defaultInstance.currencyFormatter.unformat($(this).find('.budget-table-cell-activity').text());
-                var budgeted = available + activity;
-                var burned = activity / budgeted;
-                var pace = burned / timeSpent();
-
-                let masterCategoryViewId = $(this).prevAll('.is-master-category').attr('id');
-                let masterCategory = ynabToolKit.shared.getEmberView(masterCategoryViewId).get('data');
-                let masterCategoryId = masterCategory.get('categoryId');
-                var masterCategoryDisplayName = masterCategory.get('displayName');
-
-                let subCategoryViewId = $(this).attr('id');
-                let subCategory = ynabToolKit.shared.getEmberView(subCategoryViewId).get('data');
-                let subCategoryId = subCategory.get('categoryId');
-                var subCategoryDisplayName = subCategory.get('displayName');
-
-                var transactionCount = allTransactions.filter(function (el) {
-                  return el.outflow > 0 &&
-                    el.masterCategoryId === masterCategoryId &&
-                    el.subCategoryId === subCategoryId;
-                }).length;
-
-                var temperature;
-                if (pace > 1) {
-                  temperature = 'cautious';
+              // Select all budget table rows but not the uncategorized category and not master categories.
+              $('.budget-table-row')
+                .not('.budget-table-uncategorized-transactions')
+                .not('.is-master-category')
+                .each(function () {
+                  var available = ynab.YNABSharedLib.defaultInstance.currencyFormatter.unformat($(this).find('.budget-table-cell-available').text());
```

Comment:
Not going to stop me from merging, but it'd be nice to use our new `unformat` function now that we have one.